lib/src/rules/require_trailing_comma.dart

```diff
@@ -0,0 +1,155 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/token.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+
+import '../analyzer.dart';
+
+const _desc = r'Use trailing commas for all function calls and definitions.';
+
+const _details = r'''
+
+**DO** use trailing commas for all function calls and definitions unless the
+function call or definition, from the start of the function name up to the
+closing parenthesis, fits in a single line.
+
+**GOOD:**
+```dart
+void run() {
+  method(
+    'does not fit on one line',
+    'test test test test test test test test test test test',
+  );
+}
+```
+
+**BAD:**
+```dart
+void run() {
+  method('does not fit on one line',
+      'test test test test test test test test test test test');
+}
+```
+
+**Exception:** If the final parameter is positional (vs named) and either a
```

Comment:
Based on the test cases I think this exception covers both parameters and arguments, but the text sounds more like you're talking about arguments even though you use the word 'parameter'. Consider clarifying that this applies to both.

Reply:
Updated to specify both parameter/argument.